### PR TITLE
Support Project ID

### PR DESCRIPTION
**Pull request by ts-leojoseph-r**
# Description
* Added support for `RASProjectId`
* Deprecate `RASApplicationIdentifier`
* Made sure that it is backward compatible

## Links
[MINI-2038](https://jira.rakuten-it.com/jira/browse/MINI-2038)

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

**Review by corycaywood — APPROVED** [review]
Looks good 👍  I left a note about the user guide. Please also update the change log.

File: MiniApp/Classes/MiniAppSdkConfig.swift [hunk 1]
```diff
@@ -11,13 +11,21 @@ public class MiniAppSdkConfig {
             }
         }
     }
+    @available(*, deprecated)
```
Comment (corycaywood) [inline MiniApp/Classes/MiniAppSdkConfig.swift:14]: Can you add a deprecation message that they should use `rasProjectId` instead?

Comment (ts-leojoseph-r) [reply MiniApp/Classes/MiniAppSdkConfig.swift]: Done

File: USERGUIDE.md [hunk 2]
```diff
@@ -48,13 +48,15 @@ In your project configuration .plist you should add below Key/Value :
...
 | :----                        | :----:  | :----                                                           |:----:   |:----:   |
-| RASApplicationIdentifier     | String  | `Set your MiniApp host application identifier`                  |NO       |`none`   |
+| RASProjectId     | String  | `Set your MiniApp host application project identifier`                  |NO       |`none`   |
 | RASProjectSubscriptionKey    | String  | `Set your MiniApp subscription key`                             |NO       |`none`   |
 | RMAAPIEndpoint               | String  | `Provide your own Base URL for API requests`                    |NO       |`none`   |
 | RMAHostAppUserAgentInfo      | String  | `Host app name and version info that is appended in User agent. The value specified in the plist is retrieved only at the build time.` |YES      |`none`   |
 
 If you don't want to use project settings, you have to pass this information one by one to the `Config.userDefaults` using a `Config.Key` as key:
 
+NOTE: `RASApplicationIdentifier` is deprecated
```
[the remark this answers is not in the document]
Comment (ts-leojoseph-r) [reply USERGUIDE.md]: Done

File: MiniApp/Classes/Environment.swift [hunk 3]
```diff
@@ -7,17 +7,20 @@ internal protocol EnvironmentProtocol {
...
+        subscriptionKey = "RASProjectSubscriptionKey",
+        endpoint = "RMAAPIEndpoint",
+        isTestMode = "RMAIsTestMode",
+        hostAppUserAgentInfo = "RMAHostAppUserAgentInfo"
     }
 
     let bundle: EnvironmentProtocol
 
     var customUrl: String?
+    @available(*, deprecated, message: "use customProjectId instead")
```
Comment (Climbatize) [inline MiniApp/Classes/Environment.swift:21]: please use the rename parameter :)

File: MiniApp/Classes/MiniAppSdkConfig.swift [hunk 4]
```diff
@@ -11,13 +11,21 @@ public class MiniAppSdkConfig {
             }
         }
     }
+    @available(*, deprecated, message: "use rasProjectId instead")
```
Comment (Climbatize) [inline MiniApp/Classes/MiniAppSdkConfig.swift:14]: same, we should use the rename parameter of this annotation

**Review by Climbatize — APPROVED** [review]

**Pull request merged**